### PR TITLE
feat interactive v2.0

### PR DESCRIPTION
**Pull request by axif0**
### Contributor checklist

- [X] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch
- [X] I have tested my code with the `pytest` command as directed in the [testing section of the contributing guide](https://github.com/scribe-org/Scribe-Data/blob/main/CONTRIBUTING.md#testing)

---

### Description
Added test for interactive, get
Added total and list functionalities 
Updated command messages
Added prefilled language, data-types functionalities.  

interactive2 1

### Related PR

- [Closed PR #497 ](https://github.com/scribe-org/Scribe-Data/pull/497)

### CODE REVIEW

**Comment by andrewtavis** [comment]
The work in #497 is included here, @axif0? :)

**Comment by axif0** [comment]
> The work in #497 is included here, @axif0? :)

Ya..

**Comment by andrewtavis** [comment]
No need :) Is a great idea 😊 Will review over the weekend!

**Comment by andrewtavis** [comment]
Functionality wise this is looking really great, @axif0, but can we have interactive mode be per command, rather than adding it all onto `get`? So it'd be `scribe-data total -i` for the total lexemes?

**Comment by axif0** [comment]
Modified `scribe-data get -i` and created `scribe-data total -i`.

Updated PyICU funtionality. Now user don't have to execute the command again.
image

File: src/scribe_data/cli/get.py [hunk 1]
```diff
@@ -136,7 +142,14 @@ def get_data(
     # MARK: Emojis
 
     elif data_type in {"emoji-keywords", "emoji_keywords"}:
-        generate_emoji(language=language, output_dir=output_dir)
+        if not check_if_pyicu_installed():
```
Comment (axif0) [inline src/scribe_data/cli/get.py]: By adding this user don't have to execute the command again. 
image

But I don't know why I got `403 error` and `EOFError`

**Comment by andrewtavis** [comment]
Generally things are looking good, @axif0, but I'm getting a 403 forbidden for the query from total interactive mode. Might just be the server. Could I also ask you to take a look at the [changes I made in my commit](https://github.com/scribe-org/Scribe-Data/pull/500/commits/b173d11ac84386e603119ae959734e3c50531aa5) and familiarize yourself with the comment formatting - i.e. own line comments are capitalized and punctuated, and inline comments are lower case and don't have a period :)

**Comment by axif0** [comment]
Awesome, I've reviewed the changes you mentioned. You're right, following the standard comment formatting is important. Thanks for the guidance - I appreciate you taking the time to ensure I'm aware of these best practices. I'll apply the capitalization, punctuation, and formatting standards you described in future.

**Review by andrewtavis — APPROVED** [review]
All's working now, @axif0 :) Likely just the Wikidata Query Service needing us to back off for a moment. Thanks so much for these improvements! This really has come along so well 😊

**Pull request merged**